src/store/api.js

```diff
@@ -61,9 +64,16 @@ export default ({ dispatch }) => (next) => {
           method = 'GET';
           break;
       }
+      let req;
+
+      if (action.payload.endpoint === '/predict') {
```

Comment:
@indiejoseph Should remove this part right? BTW, you can put api mock on `/mockRestServer/server.js`
like this:
```
server.post('/predictions', (req, res) => {
  res.jsonp({
    predictions: [
      {
        name: 'Rose',
        probability: 0.96,
      }, {
        name: 'Lily',
        probability: 0.12,
      },
    ],
  });
});
```